typescript/sdk/src/token/EvmERC20WarpRouteReader.ts

```diff
@@ -97,6 +97,7 @@ export class EvmERC20WarpRouteReader extends HyperlaneReader {
       [TokenType.nativeScaled]: null,
       [TokenType.collateralUri]: null,
       [TokenType.syntheticUri]: null,
+      [TokenType.collateralDex]: null,
```

Comment:
Considering that we are mapping dex collateral to be collateral on evm, shouldn't we use the same reading function here instead of just setting to null?

Reply:
it does not exist on evm, thats why I added null there

Reply:
yeah i get just found wierd that in https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/0d4fa77350cc3468ce632f3663dea9215003d1a1/typescript/sdk/src/token/contracts.ts#L50 you are mapping the type to the collateral contract and then we have the reader set to null here but not big deal